### PR TITLE
[chore] remove references to InstrumentationLibrary

### PR DESCRIPTION
**Pull request by codeboten**
This was renamed to scope, cleaning up references in this change.

### CODE REVIEW

File: receiver/riakreceiver/testdata/integration/expected.json [hunk 1]
```diff
@@ -211,4 +211,4 @@
          }
       }
    ]
-}
+}
```
Comment (dmitryax) [inline receiver/riakreceiver/testdata/integration/expected.json:214]:
```suggestion
}

```

File: receiver/riakreceiver/testdata/scraper/expected_disabled.json [hunk 2]
```diff
@@ -160,4 +160,4 @@
          }
       }
    ]
-}
+}
```
Comment (dmitryax) [inline receiver/riakreceiver/testdata/scraper/expected_disabled.json:163]:
```suggestion
}

```

**Review by dmitryax — COMMENTED** [review]
Couple nits, otherwise LGTM

**Review by dmitryax — APPROVED** [review]

**Review by mx-psi — APPROVED** [review]
LGTM. cc @gbbr this will conflict with your PR (but we can't delay it)

**Pull request merged**